lib/chef/request_id.rb

```diff
@@ -15,7 +15,6 @@
 # limitations under the License.
 #
 
-require 'securerandom'
```

Comment:
I just loaded irb and typed ```Random.new``` and it worked, then I tried ```require "secure"``` and that failed. That's why I said "looks like" :)